doc/ug/getting_started_fpga.md

```diff
@@ -36,25 +36,24 @@ $ ./meson_init.sh
 $ ninja -C build-out all
 ```
 
-Since not all FPGAs are able to fit the full design, there is a separate script that can be invoked to reduce the size of the design.
-Note that even though the ChipWhisperer CW310 board with Xilinx Kintex 7 XC7K410T FPGA is able to hold the full OpenTitan design, it currently requires the size of the design to be reduced just like for the Nexys Video FPGA board.
+Since some FPGAs other than the Xilinx Kintex 7 XC7K410T device on the ChipWhisperer CW310 board might not be able to fit the full design, there is a separate script that can be invoked to reduce the size of the design.
```

Comment:
Since we only really care about the CW310 and the Nexys Video board, we can make this statement less fuzzy, e.g.:

Only the CW310 board with the XC7K410T FPGA can fit the whole Earl Grey design.
To use the Nexys Video board, the Earl Grey design has to be modified using a script.

Reply:
I agree. That's done now. Thanks for your review @imphil !